.travis.yml

```diff
@@ -29,7 +29,7 @@ install: jabba install "$TRAVIS_JDK" && jabba use "$_" && java -Xmx32m -version
 
 jobs:
   include:
-    - script: sbt test headerCheck test:headerCheck grpcVersionSyncCheck
+    - script: sbt test headerCheck test:headerCheck grpcVersionSyncCheck scalafmtCheckAll scalafmtSbtCheck
```

Comment:
Question:

I guess the `test` is not necessary here due to we already have 2x2 matrix for tests.
WDYT?

Reply:
I think the other jobs run integration tests, and this is the only job that runs the unit tests - unless I'm misunderstanding you?

Reply:
The default Test stage has 2x2 matrix for running unit tests.
I think the `jobs include` add additional jobs to the default Test stage.
(https://docs.travis-ci.com/user/build-stages/#build-stages-and-build-matrix-expansion).

e.g. All 6.1 - 6.4 jobs run unit tests, and the 6.1 & 6.5 run the same test for same Scala version and same env (JDK version).
<img width="1083" alt="screenshot" src="https://user-images.githubusercontent.com/13405580/60357991-f1559d80-9989-11e9-89ce-cf65aba8d995.png">